### ISSUE TITLE
Include an attribute in the auto-complete results

### ISSUE THREAD

**Issue by dharake** [Smile-SA/elasticsuite#493]
Using Magento 2.1.7

This is not an issue so much as a request for information.

In the search results of auto-complete drop-down, each item has name, image, price. I would like to  be able to add the attribute "manufacturer" to the item as I have many products with the same name, but from different manufacturers. 

Any help would be most appreciated.

Thanks in advance,
Danny

**Comment by romainruaud** [comment]
Hello,

what you can do is the following : 

On a frontend/di.xml file you would add : 

```
    <type name="Smile\ElasticsuiteCatalog\Model\Autocomplete\Product\AttributeConfig">
        <arguments>
            <argument name="additionalSelectedAttributes" xsi:type="array">
                <item name="manufacturer" xsi:type="string">manufacturer</item>
                <item name="climate" xsi:type="string">climate</item>
                <item name="material" xsi:type="string">material</item>
            </argument>
        </arguments>
    </type>
```

And then override the "product.html" template to add your attributes.

Eg : 

```
<dd class="<%- data.row_class %>" id="qs-option-<%- data.index %>" role="option" class="product-item" href="<%- data.url %>">
    <div class="product-image-box">
        <img width="45px" height="45px" src="<%- data.image %>">
    </div>
    <div class="product-shop product-item">
        <div class="f-fix">
            <div class="product-primary"><div class="product-name"><%- data.title %></div>
                <div class="product-details"><%- data.manufacturer %></div>
                <div class="product-details"><%- data.climate %></div>
                <div class="product-details"><%- data.material %></div>
                <div class="product-details"><%- data.description %></div>
            </div>
            <div class="product-secondary">
                <div class="price-box">
                    <span class="regular-price"><span class="price"><%= data.price %></span></span>
                </div>
            </div>
        </div>
        <div class="clear"></div>
    </div>
    <div class="clear"></div>
</dd>
```

Bust be sure to use latest 2.3.x branch since I found a bug we just fixed in #495

**Comment by romainruaud** [comment]
I close the issue since the question has been answered, feel free to reopen it if needed.

Best regards

**Comment by dharake** [comment]
Hi @romainruaud 

Thank you very much for your feedback. I was indeed able to call the attributes into the autocomplete results by using your method. The only problem I'm facing is that it gets the id of the value and not the actual name i.e. "120" instead of "Honda".

I have tried fiddling around without much success. Is there an obvious way to do this?

Best regards

**Comment by romainruaud** [comment]
As I said, the issue with the ids instead of values has been fixed in #495 

Since the PR has been merge, you can use the 2.3.x branch for now, or wait for the next stable release.

Regards

**Comment by dharake** [comment]
My mistake, thanks again.
